src/components/VmStatus/strings.js

```diff
@@ -0,0 +1,25 @@
+export const RUNNING = 'Running';
+export const ERROR = 'Error';
+export const VM_ERROR = 'VM Error';
+export const POD_ERROR = 'Pod error';
+export const IMPORTING_ERROR = 'Import error';
+export const PENDING = 'Pending';
+export const STARTING = 'Starting';
+export const IMPORTING = 'Importing';
+export const IMPORTING_VMWARE = 'Importing (VMware)';
+export const IMPORTING_PENDING_VMWARE = 'Import Pending (VMware)';
+export const IMPORTING_ERROR_VMWARE = 'Import Error (VMware)';
+export const VIEW_POD_EVENTS = 'View pod events';
+export const VIEW_POD_DETAILS = 'View pod details';
+export const VIEW_VM_EVENTS = 'View VM events';
+export const UNKNOWN = 'Unknown';
+export const OFF = 'Off';
+export const MIGRATING = 'Migration';
+export const IMPORTING_VMWARE_MESSAGE =
+  'The virtual machine is being imported from VMWare. Disks will be converted to the libvirt format.';
+export const IMPORTING_ERROR_VMWARE_MESSAGE = 'The virtual machine could not be imported from VMWare.';
+export const IMPORTING_MESSAGE = 'The virtual machine is being imported. Disks will be copied from source image.';
+export const IMPORTING_ERROR_MESSAGE = 'The virtual machine could not be imported.';
+export const VMI_WAITING_MESSAGE = 'The virtual machine is waiting for resources to become available.';
+export const STARTING_MESSAGE = 'The virtual machine wait for state to become ready.';
```

Comment:
This looks weird. I'm not sure what it means. What is the VM waiting for? Ready state?

Reply:
@apinnick hi, 
This text is problematic because I do not have useful information to add on the abv. version "Starting".

What I want to do here is to give a "long" version of "Starting" if that makes sense ?
Basically, the VM is waiting for the readyness tests to be valid.

Does "Pod is waiting for virtual machine readyness tests to pass." sounds better ?

Reply:
Since the rest of the messages do not mention the pod, maybe best not to mention it here if we can find a workaround. What happens when the VM is 'ready'? It starts running? It migrates?

Reply:
> What happens when the VM is 'ready'? It starts running? It migrates?

It starts running :-)

Reply:
OK. I suggest this: "The virtual machine is preparing to start", if it's not important for the user to know about the readiness tests.

Reply:
:+1: :heart:

Reply:
+1 on not mentioning the pod. Can we shorten this to "The virtual machine is starting" ? Do we need to be so specific about the preparing part?

Reply:
If this message does not need to be substantially different from the "Starting" message, sure, that's fine with me.

Reply:
@apinnick @matthewcarleton hi, thanks.

Updated to "The virtual machine is starting" , attached screenshots :-)

"The virtual machine is starting"
![Screenshot_2019-05-27 VmStatusWrapper Starting – React Cosmos(1)](https://user-images.githubusercontent.com/2181522/58426069-31341700-80a4-11e9-8f73-21f401d43742.png)

"The virtual machine is preparing to start"
![Screenshot_2019-05-27 VmStatusWrapper Starting – React Cosmos](https://user-images.githubusercontent.com/2181522/58426078-35f8cb00-80a4-11e9-93f7-93f59f3f95f1.png)

Reply:
oh hmm now that I'm seeing this I take it back, the second one is probably better. It would be even nicer if we could be a bit more detailed. Is there any other information available on what is happening?
Also, do we want to say 'view pod events` or just `view events` Is this the users first exposure to the VM running on a pod? Would they understand that the pod = the VM? 
Sorry for all the nitpicks :)

Reply:
> Sorry for all the nitpicks :)

Not at all :-) 

> Is there any other information available on what is happening?

a - Starting means, we have all the necessary resources to start the VM, and is currently in the process of creating the vm's storage, net and compute resources.

b - The "fail description Ready" represent a description string we get from k8s API call, It should have some relevant information about this specific vm's state.

> Also, do we want to say 'view pod eventsor justview events` 

I copied it from https://github.com/kubevirt/web-ui-components/pull/427#issuecomment-490099773 , happy to change to what ever makes more sense :smile: 

> oh hmm now that I'm seeing this I take it back,

Yes, seeing things in context changes the view :-) I can make new screen shots with different texts so we can evaluate several options.

Reply:
Thanks @yaacov!
Could we say " This virtual machine will start shortly. Preparing storage, networking, and compute resources"
where does "View Pod events" go? That should influence what this string says IMO :)

Reply:
> Could we say " This virtual machine will start shortly. Preparing storage, networking, and compute resources"

+1 

> where does "View Pod events" go? That should influence what this string says IMO :)

It goes to the page of the "Pod" that runs the VM into the "Events" tab ...

Reply:
@matthewcarleton I like it now :-)

![Screenshot_2019-05-27 VmStatusWrapper Starting – React Cosmos](https://user-images.githubusercontent.com/2181522/58430210-2469f000-80b1-11e9-966f-f10841bbc425.png)

Reply:
Does the VM Details page include an Events tab (like our designs do)? If yes, maybe changing this text to "View VM events" and linking to that page makes the most sense. If not, then "View pod events" seems okay for now.

Reply:
> Does the VM Details page include an Events tab (like our designs do)?

I think the reason we send users to the pod, is that while vm is not yet running, the vm events is empty, ant the pod events are more useful.

@matthewcarleton @rawagner @suomiy maybe we should have two links ( or more ? ) one for each entity that makes sense in this context ?

Reply:
That makes total sense, I'd say "View pod events" then. 👍

Our design does support two actions/links in the footer (example [here](http://openshift.github.io/openshift-origin-design/web-console/4.0-designs/status/status.html#status-popovers)) but since VM events would be empty/useless in this state, just linking to pod events seems correct to me.

Reply:
+1 ya that makes sense to me. Pod events it is!

Reply:
VM events shows also launcher pod events see https://github.com/kubevirt/web-ui/blob/master/frontend/public/kubevirt/components/vm/vm-detail.jsx#L39 so we dont have link to pod events page anymore

Reply:
> so we dont have link to pod events page anymore

@rawagner should we:
1 - Convert all links to `vm events` instead of pod events ?
2 - Convert all labels from "View pod events" to just "View events" ?

Reply:
I think it makes sense to show the link to the pod at least in a case of an error - for inspecting the pod

Reply:
@rawagner @suomiy 
All links except pod-error will link to vm event and label "View events"
In case of pod-error link to pod events and label "View pod events" WDYT ?

Reply:
I would link to VM events and to Pod Overview..